crates/typst/src/math/equation.rs

```diff
@@ -204,6 +204,27 @@ impl MathParItem {
     }
 }
 
+/// Computes the origin position and the size of the bounding box that covers
+/// a list of boxes from left to right
+fn compute_bounding_box(pos_and_sizes: &[(Point, Size)]) -> (Point, Size) {
```

Comment:
This helper function should be moved below the code that uses it. In Typst the order is always general -> specific.